### ISSUE TITLE
Feedback on the current Styleguide design

### ISSUE THREAD

**Issue by quicksketch** [backdrop-ops/styleguide#11]
@dariusgarza has done some amazing work building out the styleguide in this repository. It can be previewed at http://backdrop-ops.github.io/styleguide.  This issue is to cover feedback of the current design (not implementation details)
## Blocking design issues

**Progress Bar** https://github.com/backdrop/backdrop-issues/issues/758
- Progress Bar might be a bit too blingy / psychedelic. New color?

**Fieldsets** https://github.com/backdrop/backdrop-issues/issues/755
- Fieldsets collapsed/collapsible/regular all look the same, thinking collapsed should look particularly 'clickable', as they can be hiding items the user is looking for.
- The color difference between the fieldset labels and the rest of the labels is very subtle - see [recommendations](https://github.com/backdrop-ops/styleguide/issues/11#issuecomment-75392784)

**Form elements pt 3** https://github.com/backdrop/backdrop-issues/issues/883
- Buttons look good on their own, but when a primary and secondary button exist at the same time the green-on-hover for the secondary button has a greater feeling of primary-ness than the primary button.

**Comment by quicksketch** [comment]
Overall looks amazing!

**Comment by wesruv** [comment]
Adding my two cents, fwiw:
- ~~Blockquote - worried that these styles might get a little cramped on skinny viewports? Pickin' nits though. Maybe a little more width and a teensy less padding?~~
- ~~This might be too late, but I think the &raquo; in the breadcrumbs should be added using CSS pseudo element instead of HTML, since it's presentational content (implementation detail)~~ **Moved to implementation ticket.**

**Comment by ghost** [comment]
- Missing styling on quotes? (`<q>`)
- ~~Ordered lists have more left margin than unordered lists (both should be equal, I prefer the margin amount on ordered lists)~~
- Can/should we add a subtle row highlight to tables on hover? (e.g. http://www.textfixer.com/tutorials/highlight-table-row.php)
- ~~Increase `<blockquote>` width to 90%~~
- ~~Overall size of all headings can be increased (something like 250%, 180%, 130%, 100%, 90%, 80% respectively?)~~
  -~~ `<h6>` missing color styling (is grey instead of black like other headings)~~
- ~~Monospace (`<code>`) font size can be decreased (to 1em?)~~

BTW, Love! the styling for the pager (nice to finally have decent-sized pager links).

**Comment by klonos** [comment]
On Chrome and Firefox for android that I tested, the text size difference is really big for various elements. For example...

Messages (huge text) compared to the rest of the site text:
![messages](https://cloud.githubusercontent.com/assets/2423362/6290262/badee8dc-b979-11e4-9e22-b56e117f2a08.png)

Vertical tabs labels text (tiny) compared to the tab content text:
![vtabs_labels](https://cloud.githubusercontent.com/assets/2423362/6290270/c90f1b20-b979-11e4-8f0c-43a618295972.png)

**Comment by klonos** [comment]
...BTW I just noticed that when the message text wraps to more than a single line, there is a big vertical space between the lines. You can clearly see this happening in the first screenshot in my previous comment for the warning & error messages. The reason why it is not happening for the status message is because the status message is actually an example of multiple messages of the same status merged together and the lines are ```<li>```s while the text in the warning and error messages is ```<span>```.

**Comment by quicksketch** [comment]
> the text size difference is really big for various elements. For example...

:laughing: Awesome

**Comment by wesruv** [comment]
@klonos @quicksketch
Might be good to hold off on these fixes on actual implementation instead of the style guide.

**Comment by klonos** [comment]
Fair enough.

**Comment by wesruv** [comment]
So I'm chunking out tickets for the implementation step... and got to Dialog... what is this?
![style guide backdrop](https://cloud.githubusercontent.com/assets/5607236/6311652/050a3bd2-b934-11e4-9663-a0f96b10b338.png)

**Comment by quicksketch** [comment]
@jenlampton is working on fixing the broken examples in Styleguide module. I think the dialog is busted because we get a JS error on the page currently. Jen, is that progress being tracked anywhere?

**Comment by wesruv** [comment]
Feedback from @klonos on fieldsets from https://github.com/backdrop/backdrop-issues/issues/755#issuecomment-75352191

> The color difference between the fieldset labels and the rest of the labels is very subtle. How about we do either of these:
> - intent the fieldset contents a tiny bit
> - change the label color to look like a link (something clickable)
> - move the label over the top border line of the fieldset. like the ones [here](http://www.pixy.cz/blogg/clanky/css-fieldsetandlabels.html)

**Comment by klonos** [comment]
For posterity... the "here" part in the last suggestion is a link to http://www.pixy.cz/blogg/clanky/css-fieldsetandlabels.html

**Comment by jenlampton** [comment]
@wesruv dialogs are the pop-up boxes (modals) that you see, for example, when editing a view. we'll also see them (soon) in the Token UI, when using the Rich-Text editor, and on confirm forms.  It's a common UI element used all over backdrop. Sorry it's not working in the styleguide :/

![dialog](https://cloud.githubusercontent.com/assets/397895/6518103/3d822cc2-c357-11e4-9aa5-f1a24026f730.png)

Anyone who wants to work on implementation of the Backdrop admin theme [styleguide](http://backdrop-ops.github.io/styleguide) should start by installing backdrop + and adding the [styleguide module](https://github.com/backdrop-contrib/styleguide). That will give you the correct CSS + JS as a starting point.

At some point it may be worth replacing the markup in this styleguide with the real stuff, but at this point I don't want to loose the actual style here since we are using this as our guide ;)

**Comment by wesruv** [comment]
Am I correct in thinking we should hold off on the tickets in https://github.com/backdrop/backdrop-issues/issues/736 until this has been addressed?

Can I help by fixing some of these issues, or would I be stepping on toes?

**Comment by jenlampton** [comment]
@wesruv I think it would be helpful for you to address some of these issues here. Go wild :) We can have @dariusgarza do a final review when you're done.

**Comment by wesruv** [comment]
@dariusgarza / @jenlampton I think I captured all of the Blocked/Partially blocked tickets, you can see they referenced this ticket above^^

If there's any feedback we don't plan on implementing let me know, and I can potentially unblock some of these.

**Comment by jenlampton** [comment]
I just made a few changes to the Element styles on the styleguide to unblock https://github.com/backdrop/backdrop-issues/issues/738, namely:
- blockquote is now 85% wide and has a smidge less padding
- consistent spacing and line height for OL and UL 
- increased all heading sizes (except h1.page-title)
- monospace font size is no more than 100%

**Comment by jenlampton** [comment]
I also agree with @BWPanda that we should add a subtle table row highlight, so I added a bright yellow one to the [styleguide](http://backdrop-ops.github.io/styleguide/index.html) for demonstration purposes. @dariusgarza please play with it, and if you like the highlight recommend a better color - or you can update the styleguide directly :)

**Comment by docwilmot** [comment]
Is this issue dealing with responsiveness at all? Quite a few things are breaking on small screens.

**Comment by quicksketch** [comment]
> Is this issue dealing with responsiveness at all? Quite a few things are breaking on small screens.

I think this issue should attempt to prevent any new problems on small screens, but any existing problems we have on small screens should be fixed separately in their own issues as bugs (so they'd go into the next 1.0.x release instead of 1.1.0).

**Comment by wesruv** [comment]
@docwilmot I've been testing on mobile and making sure it's looking good. That said, "mobile optimization" should probably wait until we do more of an overhaul of the admin experience.

Right now I'm shooting for "very mobile friendly" :P

**Comment by docwilmot** [comment]
Thanks. Responsiveness issue from long ago here for reference:
https://github.com/backdrop/backdrop-issues/issues/516

**Comment by jenlampton** [comment]
>  New/Updated tags, Red makes it seem like an error, feel like a label appearance (subtle bg with rounded corners) could be nice here. Green/gray/blue/etc might work better here?

I've Implemented the design suggestions for new/updated, This should un-block https://github.com/backdrop/backdrop-issues/issues/760

![mark](https://cloud.githubusercontent.com/assets/397895/7218038/282bc1f0-e60a-11e4-8243-b1253c16884b.png)

**Comment by jenlampton** [comment]
> Focus on textfields seems invisible with gray-on-gray (see firefox vs chrome)

I just updated the focus on the form input fields, so that should unblock https://github.com/backdrop/backdrop-issues/issues/753

![forms1](https://cloud.githubusercontent.com/assets/397895/7218276/e93f0112-e61a-11e4-9ff9-0dde17144196.png)
